moveit_ros/moveit_servo/config/panda_simulated_config.yaml

```diff
@@ -26,7 +27,7 @@ publish_joint_velocities: true
 publish_joint_accelerations: false
 
 ## Plugins for smoothing outgoing commands
-use_smoothing: true
+use_smoothing: false
```

Comment:
This is going to default to no smoothing at all, since `override_acceleration_scaling_factor` is zero too. Prob not what we want. Suggest reverting. Were your tests done with these settings?

Reply:
Yes, all of the hardware testing I did was with `ButterworthFilterPlugin` smoothing turned off and acceleration limits enabled. I think depending on the task, it makes sense to have it enabled. We can enable it by default.